src/Select.jsx

```diff
@@ -388,6 +389,16 @@ export default class Select extends React.Component {
       } else if (isMultipleOrTags(props) && inputValue) {
         // why not use setState?
         this.state.inputValue = this.getInputDOMNode().value = '';
+
+        if (this.props.changeOnBlur) {
```

Comment:
I think onBlur do should same as input a token separator, ref code https://github.com/react-component/select/blob/7.7.1/src/Select.jsx#L172.

use `tokenize` and only do it when tokenSeparators exist.

And I think props name use `tokenizeOnBlur` is better.

Reply:
Thank you for your feedback. I've applied the suggested changes.

Reply:
@yutingzhao1991 `tokenize` will be called as soon as input receive a token separator, so the input value will be `tokenized` before blur.

Reply:
@yesmeck Hmm... I don't understand what you mean. If there is not have a token separator in input value, we need trigger tokenize after blur.

Reply:
因为在 `onChange` 的时候已经做了 `tokenize` 了，所以 blur 的时候 inputValue 里肯定是不会又有分隔符存在的，也就不存在说只有传了 `tokenSeparators` 才在 blur 的时候做 `tokenize` 的说法。

Reply:
I see, It's the difference between `tokenizeOnBlur` and `selectOnBlur`, Seems `selectOnBlur` is better.

@sky-claudiafilipe what your option on it ?

Reply:
If we are discussing only the prop name, maybe `selectOnBlur` is more familiar. Everyone agrees?

Reply:
I think selectOnBlur is ok, and we are not only discussing the prop name. if name it `selectOnBlur`, tokenSeparators should not required.

Reply:
But we still need to call `tokenize`, correct? if we do, we will need to review `splitBySeparators` because we are doing `separators.join()` and separators can be `undefined`.  https://github.com/react-component/select/blob/785024c4dc182ed98254bfe5e3fa0f7f558cadc3/src/util.js#L137

Reply:
`const { multiple, tokenSeparators = [], children } = this.props;` I think it will works.

Reply:
Don't you prefer to do it in `defaultProps`?

Reply:
ok, seems `defaultProps` is a better way, but you should pay attention to https://github.com/react-component/select/blob/master/src/Select.jsx#L174.

Reply:
applied the changes, thank you

Reply:
Don't need call `tokenize` on `inputValue`, the `inputValue` already be tokenized in onChange. The original `findIndexInValueByLabel` is ok, but it should trigger `onSelect` as well.

What I thought is we can add it as default behavior, single select already has this behavior.

Reply:
But `onChange` only tokenizes inputValue if it `includesSeparators`, which is not true in this case. That's why I think we need to tokenize inputValue onBlur. 

Adding as default behavior, meaning remove the prop?

Reply:
You are right, I mean we don't need to call `tokenize` here again, use `findIndexInValueByLabel` as you do in the previous commit is enough.

> Adding as default behavior, meaning remove the prop?

Yes, I thought it won't break things.

Reply:
applied the changes